lib/companies_web/views/job_view.ex

```diff
@@ -17,4 +19,18 @@ defmodule CompaniesWeb.JobView do
       "input"
     end
   end
+
+  def remote_ribbon(%{remote: true}), do: "has-ribbon"
+  def remote_ribbon(_), do: ""
+
+  def link_title(url) do
+    url
+    |> String.replace("http://", "")
```

Comment:
I think we can find a way to do this ideally without three separate operations. Could we use a regex?